### PR TITLE
Merge QoS logical pipelines

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
currently there are 2 QoS pipelines for ingress (ls_in_qos_mark, ls_in_qos_meter) and egress (ls_out_qos_mark, ls_out_qos_meter). This is not necessary as there are no actions across the two pipelines that depend on each other. The two pipelines can be merged.

https://issues.redhat.com/browse/FDP-397

### CODE REVIEW

File: northd/northd.c [hunk 1]
```diff
@@ -7081,24 +7081,40 @@ build_qos(struct ovn_datapath *od, struct lflow_table *lflows,
...
+        for (size_t n = 0; n < qos->n_bandwidth; n++) {
+            if (!strcmp(qos->key_bandwidth[n], "rate")) {
+                rate = qos->value_bandwidth[n];
+            } else if (!strcmp(qos->key_bandwidth[n], "burst")) {
+                burst = qos->value_bandwidth[n];
+            }
+        }
+        if (rate) {
+            stage = ingress ? S_SWITCH_IN_QOS : S_SWITCH_OUT_QOS;
+            //ds_clear(&action);
```
Comment (almusil) [inline northd/northd.c:7107]: nit: leftover

File: tests/ovn.at [hunk 2]
```diff
@@ -37685,7 +37685,7 @@ tpa=$(ip_to_hex 10 0 0 10)
 send_garp 1 1 $eth_src $eth_dst $spa $tpa
 
 OVS_WAIT_UNTIL([test 1 = `cat hv1/ovn-controller.log | grep "pinctrl received  packet-in" | \
-grep opcode=BIND_VPORT | grep OF_Table_ID=29 | wc -l`])
+grep opcode=BIND_VPORT | grep OF_Table_ID=28 | wc -l`])
```
Comment (almusil) [inline tests/ovn.at:37688]: Seems like one place wasn't address by the table number series. Could you please use `$(ovn-debug lflow-stage-to-oftable ls_in_arp_rsp)`  instead of the 28 and submit it as separate patch?

File: tests/ovn.at [hunk 3]
```diff
@@ -10170,6 +10170,7 @@ get_final_nw_tos() {
 check_tos() {
     # First check with ovn-trace for logical flows.
     echo "checking for tos $1"
+    ovn-sbctl dump-flows lsw0
```
Comment (almusil) [inline tests/ovn.at:10173]: nit: leftover

**Review by almusil — CHANGES_REQUESTED** [review]
Not sure how to add comment to commit message (if possible), but the Jira link should be in format `Reported-at: https://issues.redhat.com/browse/FDP-397` otherwise it won't be picked up by the automation. Also I would recommend running ovs robot on this patch before submitting to ML: `./utilities/checkpatch.py -1`.

Otherwise it looks good. Thanks!

**Comment by JacobTanenbaum** [comment]
this merged from the mailing list
